app/models/validations/sales/soft_validations.rb

```diff
@@ -123,6 +123,19 @@ def discounted_ownership_value_invalid?
     mortgage_deposit_and_grant_total != value_with_discount && discounted_ownership_sale?
   end
 
+  def buyer1_not_livein?
+    return unless ownershipsch && buy1livein
+
+    (discounted_ownership_sale? || shared_ownership_scheme?) && buy1livein == 2
+  end
+
+  def buyer2_not_livein?
+    return unless ownershipsch && buy2livein
+    return unless joint_purchase?
```

Comment:
I don't think it matters, but I'm fairly confident that this line isn't necessary
since `if !joint_purchase?` then `buy2livein` is not routed to and therefore cleared.
perfectly fine to have a safety net though!